security/tests/integration/nodeAgentTest/node_agent_test.go

```diff
@@ -112,6 +112,8 @@ func TestNodeAgent(t *testing.T) {
 			continue
 		}
 
+		t.Logf("Local root certificate\n%v\nRemote root certificate\n%v", orgRootCert, rootPEM)
```

Comment:
Can we log the current timestamp as well?

Reply:
Done